client/src/components/App/App.jsx

```diff
@@ -37,7 +37,7 @@ const ShowCarousel1 = styled.div`
 	position: absolute;
 	left: 0%;
 	height: 500px;
-	width: 300px;
+	width: 1000px;
```

Comment:
should make this width reactive - otherwise different window sizes (such as having the console open) will give you issues